### ISSUE TITLE
Active TE name not syncing with webapp

### ISSUE THREAD

**Issue by bryanph** [toggl/toggl-button#1179]
Please follow this guide when creating a new issue. This speeds up the process of replicating the issue and finding a solution.

### Environment details

* Operation system: Firefox
* Extension version: Latest (after #1174)

### Expected behavior
The description should be synced and changed in the webapp as well

### Actual behavior
It doesn't get synced
![te-bug1](https://user-images.githubusercontent.com/3816712/48056124-684ae580-e1b1-11e8-8c6a-b2e8cf4072f2.gif)

### Steps to reproduce the behavior
See the GIF

### Additional details

* screenshots or screencapture

**Comment by langri-sha** [comment]
Could be it was something transient with sockets or the timer form. Worth taking another look.
